python/paddle/fluid/initializer.py

```diff
@@ -165,7 +166,8 @@ def __call__(self, var, block):
                 'force_cpu': self._force_cpu or force_init_on_cpu()
             },
             stop_gradient=True)
-        var.op = op
+        if not imperative_base.enabled():
```

Comment:
maybe use `__imperative_mode__()` to avoid potential cycle dependency

Reply:
may be fixed in next pr